l3kernel/l3text.dtx

```diff
@@ -192,9 +196,9 @@
 %
 %  For titlecasing, note that there are two functions available. The
 %  function \cs{text_titlecase:n} applies (broadly) uppercasing to the first
-%  letter of the input, then lowercasing to the remainder. In contrast,
+%  letter of each \meta{word}, then lowercasing to the remainder. In contrast,
 %  \cs{text_titlecase_first:n} \emph{only} carries out the uppercasing operation,
-%  and leaves the balance of the input unchanged. Determining whether
+%  and leaves the balance of the each \meta{word} unchanged. Determining whether
```

Comment:
Typo: "of the each <word>". (I also had a bit of trouble understanding "balance" here at first, OED says this usage is 'commerce slang'.)

Reply:
I've adjusted